tests/test_freshness_date_parser.py

```diff
@@ -1160,6 +1199,7 @@ def test_normalized_relative_dates(self, date_string, ago, period):
         param('17 सेकंड बाद', in_future={'seconds': 17}, period='day'),
         param('1 वर्ष, 5 महीने, 1 सप्ताह में',
               in_future={'years': 1, 'months': 5, 'weeks': 1}, period='week'),
+        param('1 दशक में', in_future={'years': 10}, period='year'),
```

Comment:
Could we also add a few more test cases for the Hindi version here? For e.g.

- पांच दशक बाद
- दश दशक पश्चात
- 9 दशकों मे

Reply:
@gutsytechster Done this change, Please Look into it 😊.